contracts/src/CrossChainApplications/NativeTokenBridge/IERC20TokenSource.sol

```diff
@@ -0,0 +1,39 @@
+// (c) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+// SPDX-License-Identifier: Ecosystem
+
+pragma solidity 0.8.18;
+
+/**
+ * @dev Interface that describes functionalities for a contract that will lock ERC20 tokens and send a
+ * Teleporter message to a NativeTokenDestination contract to mint native tokens on that chain.
+ */
+interface IERC20TokenSource {
+    /**
+     * @dev Emitted when ERC20 tokens are locked in this bridge contract to be bridged to the destination chain.
```

Comment:
```suggestion
     * @dev Emitted when ERC20 tokens are locked in this source contract to be transferred to the destination chain.
```
Don't remember the conclusion for if we want to avoid "bridge" term